types/options.go

```diff
@@ -197,15 +199,16 @@ func loadStoreOptions(rootless bool, rootlessUID int) (StoreOptions, error) {
 
 // UpdateOptions should be called iff container engine received a SIGHUP,
 // otherwise use DefaultStoreOptions
-func UpdateStoreOptions(rootless bool, rootlessUID int) (StoreOptions, error) {
-	storeOptions, storeError = loadStoreOptions(rootless, rootlessUID)
+func UpdateStoreOptions() (StoreOptions, error) {
+	storeOptions, storeError = loadStoreOptions(unshare.IsRootless(), unshare.GetRootlessUID())
 	return storeOptions, storeError
 }
 
 // DefaultStoreOptions returns the default storage ops for containers
-func DefaultStoreOptions(rootless bool, rootlessUID int) (StoreOptions, error) {
+func DefaultStoreOptions() (StoreOptions, error) {
+	uid := unshare.GetRootlessUID()
 	once.Do(func() {
-		storeOptions, storeError = loadStoreOptions(rootless, rootlessUID)
+		storeOptions, storeError = loadStoreOptions(uid != 0, uid)
```

Comment:
Why is `loadStoreOptions` called with differently-computed `rootless` parameter on these two code paths? If there is a non-trivial reason, please document that.

(And if the two shouldn’t differ, the values could be computed inside `loadStoreOptions` instead of the callers.)